src/main/java/com/amazon/ion/impl/IonReaderContinuableApplicationBinary.java

```diff
@@ -1006,6 +1006,10 @@ public String stringValue() {
 
     @Override
     public SymbolToken symbolValue() {
+        if (valueTid.isInlineable) {
+            return new SymbolTokenImpl(stringValue(), SymbolTable.UNKNOWN_SYMBOL_ID);
+        }
```

Comment:
🗺️ Before this change, the application reader would read an inline symbol `0x616263` as the number `0x616263` rather than the text `abc`.